tests/client/net_handler_client.h

```diff
@@ -87,4 +87,10 @@ void print_next_dev_data();
  */
 void update_tcp_output_fp(char* output_address);
 
+/**
+* Setups connection to keyboard_inputs python script. Processes incoming inputs and translates
+* it to gamepad and joystick info to send through the send_gamepad_state() function
+*/
+void setup_keyboard();
```

Comment:
I think you meant to add this to a new file `keyboard_interface.h`? I'm a little confused about the file dependencies actually.

- `net_handler_client` -- Functions to start/stop net handler, and send inputs to net handler; Depends on `net_util.h`
- `net_handler_cli` -- Prompts user input; Depends on `net_handler_client`
- `keyboard_interface` -- Opens a socket to accept connection from `keyboard_inputs.py`; Depends on `net_handler_client`

Maybe `keyboard_interface` can be put completely in `net_handler_client`, so we follow the same structure that all the CLIs gives prompts to call client functions?

Reply:
See comment above

Reply:
Hmm actually I might just end up putting it in a .h file to just clean up the stuff in there such as enums and includes